api-report/runtime-definitions.api.md

```diff
@@ -233,7 +239,7 @@ export interface IFluidDataStoreChannel extends IFluidRouter, IDisposable {
 }
 
 // @public
-export interface IFluidDataStoreContext extends IEventProvider<IFluidDataStoreContextEvents>, Partial<IProvideFluidDataStoreRegistry>, IProvideFluidHandleContext {
+export interface IFluidDataStoreContext extends IEventProvider<IFluidDataStoreContextEvents>, Partial<IProvideFluidDataStoreRegistry>, IProvideFluidHandleContext, IProvideExperimentalFluidGCInfo {
```

Comment:
since this is an object we own can't it just expose the data without a provider? what value does the provider provide. FluidObject should be used for anonymous objects, the context is never anonymous, you know what is it if you have it

I know we are using it for IProvideFluidDataStoreRegistry, but i'm not sure we should, and additionally that is just proxying our the user provided type.

Reply:
This is only here to plumb it to the DataObject. Does that answer your question?

Reply:
eventually i'd like to remove the anonymous object access completely from the fluid framework, and just let applications define it if they want it, or at least move it to the framework layer. That is probably not a near term thing, but here you are just adding and api just like any other api. if consumer want anonymous object access to that api i'd let them build it.